src/web/src/pages/layouts/MDXPageBase.tsx

```diff
@@ -23,16 +22,20 @@ const useStyles = makeStyles((theme) => {
       '& h1': {
         color: theme.palette.text.secondary,
         fontSize: 24,
-        paddingLeft: '5px',
+        paddingLeft: '150px',
```

Comment:
These work well on wide screens, but at narrower viewport sizes, you end up with a ton of whitespace and very little content.  We need some breakpoint magic here (cc @PedroFonsecaDEV, @tonyvugithub, @huynguyez):

![Screen Shot 2021-03-17 at 10 48 52 AM](https://user-images.githubusercontent.com/427398/111487175-84120f00-870e-11eb-85fc-52dd8252426f.png)